src/Sylius/Bundle/CoreBundle/spec/Sitemap/Renderer/SitemapRendererSpec.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace spec\Sylius\Bundle\CoreBundle\Sitemap\Renderer;
+ 
+use PhpSpec\ObjectBehavior;
+use Sylius\Bundle\CoreBundle\Sitemap\Model\SitemapInterface;
+use Sylius\Bundle\CoreBundle\Sitemap\Renderer\TwigAdapterInterface;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapRendererSpec extends ObjectBehavior
+{
+    function let(TwigAdapterInterface $adapter)
+    {
+        $this->beConstructedWith(
+            $adapter,
+            array(
+                'template' => '@SyliusCore/Sitemap/url_set.xml.twig',
```

Comment:
No point in splatting the array on 3 lines when there's only one entry, and remove the comma.